packages/contracts/contracts/L1/Colosseum.sol

```diff
@@ -33,22 +38,19 @@ contract Colosseum is Initializable, Semver {
      *  4) CHALLENGER_TURN    → bisect()                            → ASSERTER_TURN
      *  5) CHALLENGER_TURN    → when isAbleToBisect() returns false → READY_TO_PROVE
      *  6) CHALLENGER_TURN    → on bisection timeout                → CHALLENGER_TIMEOUT
-     *  7) ASSERTER_TIMEOUT   → when proveFault() succeeds          → PROVEN
+     *  7) ASSERTER_TIMEOUT   → when proveFault() succeeds          → NONE
```

Comment:
In this case, the challenge is deleted, so it's probably more accurate to say it's deleted rather than in a NONE state.

Reply:
IMO, since the deleted challenge can be recreated, it's logically natural for the status to cycle back to `NONE` when it's deleted.